docs/adr/0007-minimal-model.md

```diff
@@ -0,0 +1,46 @@
+# 7. Minimal Model
+
+Date: 2019-01-21
+
+## Context
+
+We currently have several classes that we think of as models.
+
+   * ActiveRecord classes that reflect persisted data in the database. These are conventionally found in `app/models`.
+   * Plain Ruby classes that reflect readonly data stored in configuration files. These currently have no place to live :-(.
+
+We also have other classes that ressemble models, but arent't e.g.
+
+   * `DocumentUrl` is not a class we think of as a model e.g. because it has no public attributes
+   * `UserFacingState` is not a class we think of as a model e.g. because its instances have no unique ID
+
+We would like to make it clear which classes belong in `app/models`.
+
+## Decision
+
+We should treat a class as a model if it has the following properties.
+
+   * **The name of the class is a noun that represents a domain concept**
+      * Example: a `Document` represents all revisions of a piece of [...]
+      * Guideline: it should be difficult to rename the class to a verb
+      * Guideline: the class should be documented in the README nomenclature
+   * **Instances of the class have multiple attributes that encapsulate it**
+      * Example: a `DocumentType` encapsulates publishing metadata [...]
```

Comment:
Does this ellipsis need to be filled out?

Reply:
Done :-).